_dbt/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13
+    restart: always
+    environment:
+      POSTGRES_DB: my_db
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+    ports: 
+      - "5432:5432"
+    volumes:
+    - ./postgres-data:/var/lib/postgressql/data
+
+  dbt-modeling:
+    container_name: dbt-modeling
+    image: ghcr.io/dbt-labs/dbt-postgres:1.8.2
+    volumes:
+      - .:/usr/app/dbt
+    ports:
+      - "8080:8080"
+    environment:
+      DBT_PROFILES_DIR: /usr/app/dbt
+      DBT_TARGET: dev
```

Comment:
I think this variable can be removed as default target is specified on the 2nd line of your profiles.yml file